### PR TITLE
Opensearch Cross Cluster Replication Setup Example via TF

### PR DESCRIPTION
**Pull request by mparikhaiven**
# Update this heading, and describe your changes

"Initial Commit for Opensearch Cross Cluster Replication Example"

### CODE REVIEW

File: solutions/opensearch-cross-cluster-reoplication/terraform/provider.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    aiven = {
+      source = "aiven/aiven"
+      version = ">= 2.5.0, < 3.0.0" }
```
Comment (runwuf) [inline solutions/opensearch-cross-cluster-reoplication/terraform/provider.tf:5]: 3.11.0 is out, perhaps test it against the new version to make sure it works and remove the constraint `3.0.0`?
https://github.com/aiven/terraform-provider-aiven/releases/tag/v3.11.0

**Review by runwuf — COMMENTED** [review]
minor comment on terraform version, otherwise LGTM

File: solutions/opensearch-cross-cluster-reoplication/terraform/provider.tf (hunk 1 above)
Comment (mparikhaiven) [reply solutions/opensearch-cross-cluster-reoplication/terraform/provider.tf]: @runwuf - Updated and tested, can u pls check now? thnx

**Review by runwuf — APPROVED** [review]

**Review by runwuf — CHANGES_REQUESTED** [review]
one more thing, I would suggest to `git mv` this to `/terraform/opensearch/`

**Review by runwuf — APPROVED** [review]

**Pull request merged**